index.js

```diff
@@ -31,12 +31,6 @@ function HtmlWebpackPlugin(options) {
   if(this.options.template.indexOf('!') === -1) {
     this.options.template = require.resolve('./loader.js') + '!' + path.resolve(this.options.template);
   }
-  // Resolve template path
```

Comment:
Okay, but these lines are not tested. Could you maybe write some tests for that scenario? I have no idea how to write a test for that usecase.